README.md

```diff
@@ -35,20 +35,48 @@ make build IMAGE=myregistry.local/iop-gateway:dev
 ```
 
 ### Running the Container Locally
+
+Before running the container make sure to create TLS certificates using:
+```bash
+make certs
+```
+
 After building, you can run the container using Podman:
 
 ```bash
-podman run --rm -p 8080:8080 myregistry.local/iop-gateway:dev
+podman run --rm -p 8080:3000 -p 8443:8443  -v ./certs/:/etc/nginx/certs:Z myregistry.local/iop-gateway:dev
 ```
 
-You can also mount custom configuration:
+You can also mount custom configuration that would be included in the `http` block:
 
 ```bash
-podman run --rm -p 8080:8080 -v $(pwd)/config:/etc/nginx/:Z myregistry.local/iop-gateway:dev
+podman run --rm -p 8080:3000 -p 8443:8443 -v $(pwd)/config:/etc/nginx/conf.d/:Z myregistry.local/iop-gateway:dev
 ```
 
+### TLS
+
+The TLS is opened on port 8443.
+
+Running the gateway requires the following certificates:
+
+| Certificate path                    | Type                                  |
+| :---------------------------------- | :------------------------------------ |
+| `/etc/nginx/certs/candlepin_ca.pem` | CA PEM certificate (from Candlepin)   |
```

Comment:
This should just be generic, and not mention Candlepin. Even in a Foreman or Satellite deployment, the CA that would be used is not from Candlepin, it is generated by other tooling.